Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -109,11 +109,11 @@ def test_bottle
     formula_file = CoreFormulaRepository.new.formula_dir/"testball.rb"
     formula_file.write <<-EOS.undent
       class Testball < Formula
-        url "https://example.com/testabll-0.1.tar.gz"
+        url "https://example.com/testball-0.1.tar.gz"
       end
     EOS
     HOMEBREW_CACHE.cd do
-      assert_match /testball-0\.1.*\.bottle\.tar\.gz/,
+      assert_match %r{testball-0\.1.*\.bottle\.tar\.gz},
```

Comment:
Thanks and wow, that's a weird one! The warning from the parser is quite misleading. Ruby 2.2+ is a bit more helpful by printing (`ruby -wc <file>`):

> `warning: ambiguous first argument; put parentheses or a space even after '/' operator`

But when actually executing the code, the ambiguity is suddenly gone and everything is fine. :confused: My preference would be to put parentheses around the arguments of `assert_match` instead, but I can understand why one wouldn't want to do this, so I'm not opposing the `%r{` solution.